### ISSUE TITLE
Failed to execute goal on project iguana.datagenerator.

### ISSUE THREAD

**Issue by JessicaAlphy** [dice-group/IGUANA#82]
I am installing the Iguana ResultProcessing Module using the guideline on the website: **https://github.com/dice-group/IGUANA/tree/master/iguana.corecontroller** . When I type the order : `mvn clean package -DskipTests` , some error occurred, which are shown as follows:

`[ERROR] Failed to execute goal on project iguana.datagenerator: Could not resolve dependencies for project org.aksw:iguana.datagenerator:jar:2.0.0: Failure to find org.aksw:iguana.commons:jar:2.0.1 in http://maven.aliyun.com/nexus/content/groups/public was cached in the local repository, resolution will not be reattempted until the update interval of nexus-aliyun has elapsed or updates are forced.`

Could you please give me a hand, many thanks.

**Comment by TortugaAttack** [comment]
hey, 
you need to execute `mvn clean package -DskipTests` in the main folder. It then installs the needed packages automatically locally into the maven repo. 

If you just want to execute Iguana you can use the release jars: https://github.com/dice-group/IGUANA/releases/tag/v2.1.0c 
they can be started using: 
`java -cp iguana.resultprocessor-2.0.0.jar org.aksw.iguana.rp.controller.MainController` 
(https://github.com/dice-group/IGUANA/wiki/Installation#using-download-1)
and 
`java -cp iguana.corecontroller-2.0.0.jar org.aksw.iguana.cc.controller.MainController `
(https://github.com/dice-group/IGUANA/wiki/Installation#using-download-2)

Hope this helps. 
Cheerz :)

**Comment by JessicaAlphy** [comment]
Thank you so much, I indeed execute `mvn clean package -DskipTests` in the main folder, i.e., `[root@cll IGUANA-master]# ` , but the error still occurs: 
```
[ERROR] Failed to execute goal on project iguana.datagenerator: Could not resolve dependencies for
 project org.aksw:iguana.datagenerator:jar:2.0.0: Failure to find org.aksw:iguana.commons:jar:2.0.1 in 
http://maven.aliyun.com/nexus/content/groups/public was cached in the local repository, resolution will
 not be reattempted until the update interval of nexus-aliyun has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :iguana.datagenerator
```

**Comment by TortugaAttack** [comment]
Regarding the issue with the maven caching:
this may help: https://stackoverflow.com/questions/4856307/when-maven-says-resolution-will-not-be-reattempted-until-the-update-interval-of

**Comment by JessicaAlphy** [comment]
Thank you for your response, the abovementioned issue has been addressed. 
 Now, I am importing the project into the eclipse on the windows OS. The project can be built success. But I have no idea:
① How to run the project, (call MainController.java? or DataGeneratorController.java)
I attemptted to run both the two classes, but no result shows in the console. :( .
② Which configuration file I need to set, where to set the input parameter? (iguana.controller.src.main.resources.iguana.properties file? ) where to write the following settings:

```
<random-function class="org.aksw.iguana.generate.DatasetGeneratorImpl" generate="true" initFile="dataset100.nt" roh="0.7" ch="0.7">
			<percent value="1.0" file-name="dataset100.nt" /> <!-- only if you want to test too--> 
			<percent value="0.5" file-name="dataset50.nt" />
			<percent value="0.2" file-name="dataset20.nt" />
			<percent value="0.1" file-name="dataset10.nt" />
</random-function>
```
③ Which main class should be called， if the DataGeneratorController.java should be invoked first, how to set the input arguments?
Thank you in advance.
